pocs/scheduler/observation.py

```diff
@@ -157,10 +164,10 @@ def last_exposure(self):
 ##################################################################################################
 
     def reset(self):
```

Comment:
It would be great if this method were called by `__init__`, so that they can't get out of sync.